test/integration/patients/patient/patient-flow.js

```diff
@@ -290,9 +290,14 @@ context('patient flow page', function() {
 
     cy
       .routeAction(fx => {
-        fx.data.id = '2';
-        fx.data.relationships.state = { data: { id: '22222' } };
-        fx.data.relationships.form = { data: { id: '1' } };
+        fx.data = getAction({
+          id: '2',
```

Comment:
can we not set the id here too

Reply:
Same comment as above. Think we need to set some sort of it here because of the `PATCH` request this test is making.

Reply:
the patch should use the id returned from the `getAction`

Reply:
Updated in https://github.com/RoundingWell/care-ops-frontend/pull/1249/commits/e422b40b6a59a5afa856450c31590c79b3781071.